### PR TITLE
[Phase 1] Remove Stream and Optional usages for module: portal-search-web part 2

### PR DESCRIPTION
**Pull request by julschong**
@tinatian Hi Tina,
this pull follows the common method patent in this module as we discussed yesterday

Naming convention:
- `getValueOptional returning Optional of value`
  is changed to
  `getValue returning Value or null`

- `getValueString returning Value or Blank`
  unchanged

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:search

**Comment by Kvale1733** [comment]
@julschong No regressions.

**Comment by julschong** [comment]
@dantewang Hi Dante, 
if you have time today, could you help me review this PR?
Thank!

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/helper/PortletPreferencesHelper.java [hunk 1]
```diff
@@ -34,31 +35,43 @@ public PortletPreferencesHelper(
...
+
+		return value;
 	}
 
-	private Optional<String> _getValue(String key) {
-		return _portletPreferencesOptional.flatMap(
-			portletPreferences -> SearchStringUtil.maybe(
-				portletPreferences.getValue(key, StringPool.BLANK)));
+	private String _getValue(String key) {
+		if (!_portletPreferencesOptional.isPresent()) {
```
Comment (dantewang) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/helper/PortletPreferencesHelper.java:60]: @julschong Please make this field non-optional, and just do `field = optional.orElse(null)` in the constructor. Then when you modify the api later you can simply touch the constructor.

File: modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/helper/PortletPreferencesHelper.java [hunk 2]
```diff
@@ -30,49 +31,46 @@ public class PortletPreferencesHelper {
...
+
+		if (Validator.isBlank(value)) {
+			return null;
+		}
+
+		return value;
 	}
 
-	private final Optional<PortletPreferences> _portletPreferencesOptional;
+	private final PortletPreferences _portletPreferences;
```
Comment (julschong) [inline modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/helper/PortletPreferencesHelper.java:74]: pushed and changed here

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:search

**Comment by Kvale1733** [comment]
ci:stop

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:search

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by dantewang** [comment]
ci:forward:force

cc @shuyangzhou @tinatian

**Pull request closed without merging**